src/js/entities-service/entities/forms.js

```diff
@@ -48,7 +54,7 @@ const _Model = BaseModel.extend({
     return get(this.get('options'), 'beforeSubmit', defaultBeforeSubmit);
   },
   getSubmitReducers() {
-    return get(this.get('options'), 'submitReducers', []);
+    return get(this.get('options'), 'submitReducers', [defaultSubmitReducer]);
```

Comment:
See my comment [here](https://github.com/RoundingWell/care-ops-forms/pull/1275/commits/133b240e1581dc49c84cd30cb7ead8d8329eda86#r1394773881). I'm not sure where it is best to handle this use case, but `defaultSubmitReducer` won't run on forms that are deployed from the form builder branch, even if they do not have any `submitReducers` added.